internal/sli/query/query_processing.go

```diff
@@ -89,7 +90,7 @@ func (p *Processing) executeUSQLQuery(metricsQuery string, startUnix time.Time,
 
 	requestedDimensionName := querySplits[2]
 	if requestedDimensionName == "" && resultType != resultTypeSingleValue {
-		return 0, fmt.Errorf("USQL dimension must not be empty, unless you use %s result type", resultTypeSingleValue)
+		return 0, fmt.Errorf("USQL dimension should not be empty, unless you use %s result type", resultTypeSingleValue)
```

Comment:
Would it be possible to change this to `"USQL dimension should not be empty unless result type is %s"`?